gradle/scripts/defaultBuildProperties.gradle

```diff
@@ -28,7 +28,7 @@ def BuildProperties BUILD_PROPERTIES = new BuildProperties(project)
     .register(new BuildProperty("doNotSignArtifacts", false, "Do not sight Maven artifacts"))
     .register(new BuildProperty("gobblinFlavor", "standard", "Build flavor (see http://gobblin.readthedocs.io/en/latest/developer-guide/GobblinModules/)"))
     .register(new BuildProperty("hadoopVersion", "2.3.0", "Hadoop dependencies version"))
-    .register(new BuildProperty("hiveVersion", "1.0.1", "Hive dependencies version"))
+    .register(new BuildProperty("hiveVersion", "1.1.0", "Hive dependencies version"))
```

Comment:
Need to update Hive version for this patch: https://issues.apache.org/jira/browse/HIVE-7868

Previously, if there was an invalid schema, AvroSerDe would return a dummy error schema, and SerDeUtils.initializeSerDe() would return successfully. Now SerDeUtils.initializeSerDe throws a SerDeException instead.